scripts/dotnet.py

```diff
@@ -854,7 +854,7 @@ def __process_arguments(args: list):
         dest='channels',
         required=False,
         nargs='+',
-        default=['master'],
+        default=['main'],
```

Comment:
This is the channel for the `dotnet-install` script, yes? Did they also change that?

Reply:
I added a new channel mapping that goes from main -> tfm: net6.0, channel: master